etc/kayobe/pulp.yml

```diff
@@ -226,232 +208,7 @@ stackhpc_pulp_repo_rocky_9_highavailability_version: "{{ lookup('vars', 'stackhp
-
-  - name: CentOS Stream 8 - NFV OpenvSwitch
-    url: "{{ stackhpc_release_pulp_content_url }}/centos/8-stream/nfv/x86_64/openvswitch-2/{{ stackhpc_pulp_repo_centos_stream_8_nfv_openvswitch_version }}"
-    remote_username: "{{ stackhpc_release_pulp_username }}"
-    remote_password: "{{ stackhpc_release_pulp_password }}"
-    client_cert: ""
-    client_key: ""
-    policy: on_demand
-    proxy_url: "{{ pulp_proxy_url }}"
-    sync_policy: mirror_complete
-    state: present
-    required: "{{ stackhpc_pulp_sync_for_local_container_build | bool and stackhpc_pulp_sync_el_8 | bool }}"
-
-  - name: CentOS Stream 8 - PowerTools
-    url: "{{ stackhpc_release_pulp_content_url }}/centos/8-stream/PowerTools/x86_64/os/{{ stackhpc_pulp_repo_centos_stream_8_powertools_version }}"
-    remote_username: "{{ stackhpc_release_pulp_username }}"
-    remote_password: "{{ stackhpc_release_pulp_password }}"
-    client_cert: ""
-    client_key: ""
-    policy: on_demand
-    proxy_url: "{{ pulp_proxy_url }}"
-    sync_policy: mirror_complete
-    state: present
-    required: "{{ stackhpc_pulp_sync_for_local_container_build | bool and stackhpc_pulp_sync_el_8 | bool }}"
-
-  - name: CentOS 8 - OpsTools - collectd
-    url: "{{ stackhpc_release_pulp_content_url }}/centos/8-stream/opstools/x86_64/collectd-5/{{ stackhpc_pulp_repo_centos_stream_8_opstools_version }}"
-    remote_username: "{{ stackhpc_release_pulp_username }}"
-    remote_password: "{{ stackhpc_release_pulp_password }}"
-    client_cert: ""
-    client_key: ""
-    policy: on_demand
-    proxy_url: "{{ pulp_proxy_url }}"
-    sync_policy: mirror_complete
-    state: present
-    required: "{{ stackhpc_pulp_sync_for_local_container_build | bool and stackhpc_pulp_sync_el_8 | bool }}"
-
-  - name: CentOS Stream 8 - Ceph Pacific
-    url: "{{ stackhpc_release_pulp_content_url }}/centos/8-stream/storage/x86_64/ceph-pacific/{{ stackhpc_pulp_repo_centos_stream_8_storage_ceph_pacific_version }}"
-    remote_username: "{{ stackhpc_release_pulp_username }}"
-    remote_password: "{{ stackhpc_release_pulp_password }}"
-    client_cert: ""
-    client_key: ""
-    policy: on_demand
-    proxy_url: "{{ pulp_proxy_url }}"
-    sync_policy: mirror_complete
-    state: present
-    required: "{{ stackhpc_pulp_sync_for_local_container_build | bool and stackhpc_pulp_sync_el_8 | bool }}"
-
-  - name: ELK repository for 7.x packages
-    url: "{{ stackhpc_release_pulp_content_url }}/elasticsearch/oss-7.x/{{ stackhpc_pulp_repo_elasticsearch_logstash_kibana_7_x_version }}"
-    remote_username: "{{ stackhpc_release_pulp_username }}"
-    remote_password: "{{ stackhpc_release_pulp_password }}"
-    client_cert: ""
-    client_key: ""
-    policy: on_demand
-    proxy_url: "{{ pulp_proxy_url }}"
-    sync_policy: mirror_complete
-    state: present
-    required: "{{ stackhpc_pulp_sync_for_local_container_build | bool and stackhpc_pulp_sync_el_8 | bool }}"
-
-  - name: Grafana
```

Comment:
Looks like we've lost Grafana.

Reply:
Ah, yes. I think this is one of those things I missed then fixed in a later commit. I'll correct it here

Reply:
Fixed